frontend/public/locales/en/common.json

```diff
@@ -45,5 +45,6 @@
   "add": "Add",
   "member": "Member",
   "role": "Role",
-  "YouDoNotHavePermissionToAccessThisPage": "You don't have permission to access this page"
-}
+  "YouDoNotHavePermissionToAccessThisPage": "You don't have permission to access this page",
+  "youDoNotHavePermissionToPreformThisAction": "You don't have permission to preform this action"
```

Comment:
Change `preform` => `perform`.

I would prefer these to have shorter names like:
`no_permission_page`
`no_permission_action`